class.jetpack-cli.php

```diff
@@ -304,8 +305,43 @@ public function reset( $args, $assoc_args ) {
 				WP_CLI::success( __( 'Modules reset to default.', 'jetpack' ) );
 				break;
 			case 'prompt':
-				WP_CLI::error( __( 'Please specify if you would like to reset your options, or modules', 'jetpack' ) );
+				WP_CLI::error( __( 'Please specify if you would like to reset your options, modules or sync-checksum', 'jetpack' ) );
 				break;
+			case 'sync-checksum':
+				global $wpdb;
+				$option = 'jetpack_callables_sync_checksum';
+				if ( is_multisite() && function_exists( 'get_sites' ) ) {
+					$sites = get_sites( array( 'number' => 1000 ) );
```

Comment:
We're changing this from 100 to 1000 - is 1000 considered to cover all sites, or is there any other rationale behind it?

Reply:
Yes 1000 is suppose to cover all sites. Of course if there are larger networks this will not work.

Reply:
There is a wp_is_large_network function to check for cases of over 10k sites or users ( https://developer.wordpress.org/reference/functions/wp_is_large_network/ ). Not sure if we want to try going up that big here.

Reply:
Let's keep it at 1000 for now - if the need arises, we can increase on a case by case basis, and bump the limit in a future release.